hydromt_wflow/wflow.py

```diff
@@ -1251,7 +1261,7 @@ def setup_gauges(
             kwargs.update(crs=code)
             gdf_gauges = self.data_catalog.get_geodataframe(
                 gauges_fn,
-                geom=self.basins,
+                geom=self.basins_highres,
```

Comment:
For everywhere where you replaced with basins_highres, you should actually replace with basins.
This is a much better match. Imagine you have a gauge at high res that is just out of the basin but is included in the low res. Or for areamap, you may get nodata cell just because your cut is not the exact cut. I think we have buffers by default to in the get_data method to always read a little bit more anyhow but using basins is more consistent.

Something else, not all models will have a basins_highres geoms ie model build with a previous hydromt-wflow version and models built without hydromt. The last ones will always have basins as if basins is not in geoms then it is derived from staticmaps wflow_subcatch.

Reply:
Yeah, I was not quite sure which methods were depended on the high res map (as all old functions essentially used that). 
Thanks for the clarification @hboisgon!